mobile/lib/modules/search/ui/explore_grid.dart

```diff
@@ -56,9 +56,7 @@ class ExploreGrid extends StatelessWidget {
                       personName: content.label,
                     ),
                   )
-                : context.pushRoute(
-                    SearchResultRoute(searchTerm: 'm:${content.label}'),
-                  );
```

Comment:
This stops the handling where tapping a curated place will display all assets from its location from working.

Reply:
Correct, Also plan to fix it in a subsequent pr as well

Reply:
actually fixed it in this PR as well